acai-ios/CollectionViewCells/HeaderImageCollectionViewCell.swift

```diff
@@ -6,22 +6,23 @@
 //  Copyright © 2019 Cornell AppDev. All rights reserved.
 //
 
-import UIKit
 import SnapKit
+import UIKit
 
 class HeaderImageCollectionViewCell: UICollectionViewCell {
-    
+
     // MARK: View vars
     var imageView: UIImageView!
-    
+
     // MARK: Constraint Constants
     private enum FileConstants {
         static let imageViewHeightWidth: CGFloat = 206
+        static let imageViewTopOffset: CGFloat = 15
```

Comment:
Is this used in another location from setupConstraints? If not, leave as a let statement in that function